src/pages/apply.js

```diff
@@ -0,0 +1,1066 @@
+        </ErrorDiv>
+      </StyledLabel>
+      <StyledLabel>
+        <span>
+          Project name <Required>*</Required>
+        </span>
+        <div>
+          <small>
+            This should be a concise description of the title of your project.
+          </small>
+        </div>
+        <Input
+          type="text"
+          name="projectName"
+          value={formState.projectName.value}
+          onChange={handleInputChange}
+          maxLength="255"
+          onBlur={handleTouched}
+          required
+        />
+        <ErrorDiv>
+          {formState.projectName.isTouched &&
+            !formState.projectName.isValid && <RequiredError />}
+        </ErrorDiv>
+      </StyledLabel>
+      <StyledLabel>
+        <span>
+          Please describe what your idea/project is. <Required>*</Required>
+        </span>
+        <div>
+          <small>
+            Please describe your innovative idea, the impact that this project
+            has on Ethereum reaching the next billion, how Ethereum is/will be
+            used and the problem you’re trying to address. Do indicate the name
+            of the project, how long you have been working on this (if
+            applicable) and with whom.
+          </small>
+        </div>
+        <TextArea
+          name="projectDescription"
+          value={formState.projectDescription.value}
+          onChange={handleInputChange}
+          onBlur={handleTouched}
+          required
+        />
+        <ErrorDiv>
+          {formState.projectDescription.isTouched &&
+            !formState.projectDescription.isValid && <RequiredError />}
+        </ErrorDiv>
+      </StyledLabel>
+      <StyledLabel>
+        <span>
+          Please provide other projects or research papers (ideally public
+          and/or open-source), and engagements. If your project has an
+          open-source code repository, or if you want to share other open-source
+          contributions you’ve made please include it.
+        </span>
+        <div>
+          <small>Any links for us to check out?</small>
+        </div>
+        <Input
+          type="text"
+          name="projectPreviousWork"
+          value={formState.projectPreviousWork?.value}
+          onChange={handleInputChange}
+          maxLength="255"
+          onBlur={handleTouched}
+        />
+        <ErrorDiv>
+          {formState.projectPreviousWork.isTouched &&
+            !formState.projectPreviousWork.isValid && <UrlError />}
+        </ErrorDiv>
+      </StyledLabel>
+      <StyledLabel>
+        <span>
+          Why are you the right person to carry out this project? What makes you
+          unique? <Required>*</Required>
+        </span>
+        <div>
+          <small>(max 300 words)</small>
+        </div>
+        <Input
+          type="text"
+          name="projectLeaderReasons"
+          value={formState.projectLeaderReasons?.value}
+          onChange={handleInputChange}
+          maxLength="255"
```

Comment:
255 character limit for 300 words? 🤔

Reply:
Uhm yes, I copy pasted the 300 words from the spreadsheet. Should be 255 in the help text. Will confirm this with Sara